src/e2e-test/features/bigquery/source/BigQueryToBigQuery_withConnections.feature

```diff
@@ -0,0 +1,108 @@
+@BigQuery_Source
+Feature: BigQuery source - Verification of BigQuery to BigQuery successful data transfer using connections
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST @BQ_CONNECTION
+  Scenario: To verify data transfer from BigQuery to BigQuery with pipeline created from wrangler connection
```

Comment:
nit: `To verify data transfer from BigQuery to BigQuery with pipeline connection created from wrangler`

Reply:
updated description